app/src/lib/stores/accounts-store.ts

```diff
@@ -97,7 +97,11 @@ export class AccountsStore extends TypedBaseStore<ReadonlyArray<Account>> {
       if (__DARWIN__ && isKeyChainError(e)) {
         this.emitError(
           new Error(
-            `GitHub Desktop was unable to store the account token in the keychain. Please check you have unlocked access to the 'login' keychain.`
+            `GitHub Desktop was unable to store the account token in the keychain. 
+             Please check you have unlocked access to the 'login' keychain. Further documention can be found {' '}
+             <LinkButton onClick={this.props.onShowExamples}>
```

Comment:
Because we're inside the `AccountsStore`, where we're not allowed to emit JSX like this, I think my preferred approach to address this would be something like this:

 - create a new error type to represent this, e.g `KeyChainError`
 - throw this error instead of a regular `Error`
 - add error handling to our error handling infrastructure to detect this 
 - if we find this `KeyChainError`, we can display a popup with more context, rather than the default popup that is shown currently

For reference, our `ExternalEditorError` feels similar to this flow, so I'll point out the relevant parts:

 - created a custom error type (not sure if you need any extra fields)

https://github.com/desktop/desktop/blob/fcbb95d9df8741ab48f0dd06625c8204aa743e7c/app/src/lib/editors/shared.ts#L50-L59

 - write an error handler to look for this error (and display a popup if found)

https://github.com/desktop/desktop/blob/fcbb95d9df8741ab48f0dd06625c8204aa743e7c/app/src/ui/dispatcher/error-handlers.ts#L173-L192

 - handle the popup value to display a dialog

https://github.com/desktop/desktop/blob/fcbb95d9df8741ab48f0dd06625c8204aa743e7c/app/src/ui/app.tsx#L1490-L1503

 - wire up the error handler in the app setup:

https://github.com/desktop/desktop/blob/fcbb95d9df8741ab48f0dd06625c8204aa743e7c/app/src/ui/index.tsx#L255